### PR TITLE
Create delete comment resolver - 2

### PR DESCRIPTION
**Pull request by flacial**
Related to #893 
Updates, `submissions` queries to include the comment ID, snapshots, and GraphQL index.

### CODE REVIEW

**Comment by songz** [comment]
why are there snapshot changes that are seemingly unrelated to the actual changes?

**Review by mino323 — APPROVED** [review]

**Comment by flacial** [comment]
> why are there snapshot changes that are seemingly unrelated to the actual changes?

I think because the updated packages in the `package.json`. Before updating them, 1 snapshots, after updating them, 10 snapshots.

File: graphql/queries/getPreviousSubmissions.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ const GET_PREVIOUS_SUBMISSIONS = gql`
         name
       }
       comments {
+        id
```
Comment (anthonykhoa) [inline graphql/queries/getPreviousSubmissions.ts:26]: I think its outside the scope of this PR, but just food for thought: I think c0d3 codebase should look into refactoring queries/mutations to use fragments. We could probably save a lot of duplication by refactoring to use fragments.

https://graphql.org/learn/queries/
https://www.apollographql.com/docs/react/data/fragments/

**Review by SahilKalra98 — APPROVED** [review]

File: graphql/queries/getPreviousSubmissions.ts (hunk 1 above)
Comment (songz) [reply graphql/queries/getPreviousSubmissions.ts:26]: thanks @anthonykhoa ! Great suggestion!

https://github.com/garageScript/c0d3-app/issues/1441

**Review by songz — APPROVED** [review]

**Review by anthonykhoa — APPROVED** [review]

**Pull request merged**